src/test/crimson/seastore/onode_tree/test_fltree_onode_manager.cc

```diff
@@ -84,16 +84,18 @@ struct fltree_onode_manager_test_t
     ).then([this] {
       return tm->mount(
       ).safe_then([this] {
-	return seastar::do_with(
-	  create_mutate_transaction(),
-	  [this](auto &ref_t) {
-	    return with_trans_intr(*ref_t, [&](auto &t) {
-	      return manager->mkfs(t
-	      ).si_then([this, &t] {
-	        return submit_transaction_fut(t);
+	return repeat_eagain([this] {
```

Comment:
I'm fine with this change and the one in seastore.cc, but are they relevant to the bug described in the commit message?

Reply:
Yes, with that bug fixed, segment_cleaner starts working right after its mount, so there are chances that the cleaner transaction conflicts with mutate transactions

Reply:
Ah, move them into another commit then.

Reply:
ok